back/flag-service/.eslintrc.js

```diff
@@ -20,6 +20,8 @@ module.exports = {
     '@typescript-eslint/explicit-module-boundary-types': 'off',
     '@typescript-eslint/no-explicit-any': 'off',
     '@typescript-eslint/no-unused-vars': 'error',
+    "indent": ["error", 2, { "FunctionExpression": { "body": 1, "parameters": 2 } }],
+    "object-curly-spacing": ["error", "always"],
```

Comment:
legibility/consistency: use single quotes instead of double quotes to match the rest of the file (also change lines 26 to 29).